src/chip/src/components/Chip.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+import React from 'react';
+
+import { Chip as MuiChip, makeStyles } from '@material-ui/core';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTimes } from '@fortawesome/pro-light-svg-icons';
+import Colors from './colors';
+import type { ChipProps } from './types';
+
+const DeleteIcon = <FontAwesomeIcon fixedWidth icon={faTimes} />;
+
+const actionStates = (backgroundColor) => ({
+  '&:hover, &:focus': {
+    backgroundColor
+  },
+  '&:active': {
+    backgroundColor
+  },
+});
+
+const useStyles = (color) => {
+  const colors = Colors[color] || {};
+  return makeStyles({
+    root: {
+      ...colors
+    },
+    clickable: {
+      cursor: 'pointer',
+      ...actionStates(colors.clickable)
+    },
+    deletable: actionStates(colors.background),
+    deleteIcon: {
+      color: colors.color,
+      margin: '0 12px 0 -6px',
+    }
+  });
+};
+
+const Chip = ({
+  avatar,
+  classes,
+  clickable,
+  color = 'default',
+  component,
+  deleteIcon = DeleteIcon,
+  disabled,
+  icon,
+  label,
+  onDelete,
+  size = 'medium',
+  variant = 'default',
+  forwardRef,
+  ...rest
+} :ChipProps) => {
+  const styles = useStyles(color)();
+
+  const props :Object = {
+    ...rest,
+    avatar,
+    classes: classes || styles,
+    clickable,
+    component,
+    deleteIcon,
+    disabled,
+    icon,
+    label,
+    onDelete,
+    size,
+    variant,
+  };
+
+  if (color === 'default' || color === 'primary' || color === 'secondary') props.color = color;
```

Comment:
What does this line do? It doesn't seem like this has any effect when running it locally.

Reply:
This prevents console errors from passing invalid color prop to mui chip.